t/t5319-multi-pack-index.sh

```diff
@@ -413,6 +455,22 @@ test_expect_success 'expire does not remove any packs' '
 	)
 '
 
+test_expect_success 'expire progress off for redirected stderr' '
+	(
+		cd dup &&
+		git multi-pack-index expire 2>err &&
+		test_line_count = 0 err
+	)
+'
+
+test_expect_success 'expire with the --no-progress option' '
```

Comment:
@derrickstolee I added tests here for `expire` (and above for `write`) with the idea that if progress indicators are added for `expire` and `write` in the future these tests will validate that stderr redirection and `--no-progress` are working as expected.

Let me know if you think it's worth adding these tests now and if there are any other changes you'd recommend to the test cases (these tests are making an assumption that the command being called *would* produce a progress indicator if/when support for one is added in the future).

Reply:
Yeah, I would drop these tests for now until we add progress indicators for those subcommands.